lib/producer.ts

```diff
@@ -273,6 +277,17 @@ export class Producer implements QueueProducer {
 	): Promise<ActionRequestContract> {
 		const request = await this.storeRequest(actor, session, options);
 
+		let jobName = 'enqueue-action-request';
+		const jobParameters: string[] = [`'actionRequest'`, '$1'];
+		const values: any[] = [request];
+
+		// Handle scheduled actions
+		if (options.schedule) {
+			jobName = `enqueue-action-request-${uuidv4()}`;
```

Comment:
Need to have unique job names to avoid collisions, which cause errors at the Postgres level.